src/program/downloaders/__init__.py

```diff
@@ -107,6 +107,14 @@ def download(self, item, active_stream: dict) -> str:
         update_item_attributes(item, torrent_names)
         logger.log("DEBRID", f"Downloaded {item.log_string} from '{item.active_stream['name']}' [{item.active_stream['infohash']}]")
 
+    def add_torrent_magnet(self, magnet_link: str):
+        return self.service.add_torrent_magnet(magnet_link)
+
+
+    def get_torrent_info(self, torrent_id: str):
+        return self.service.get_torrent_info(torrent_id)
```

Comment:
These helpers are here so nothing needs to call through to any concrete downloader implementation.